src/transformers/test.py

```diff
@@ -0,0 +1,19 @@
+import datetime
```

Comment:
is this a left over file? Or you want to include this?

Reply:
It is bogus. I tried to untangle different changes from a branch of mine and this is what I get ;)